packages/transaction-controller/src/utils/first-time-interaction-api.ts

```diff
@@ -0,0 +1,75 @@
+import { createModuleLogger, type Hex } from '@metamask/utils';
+
+import { FirstTimeInteractionError } from '../errors';
+import { projectLogger } from '../logger';
+
+export const BASE_URL = 'https://accounts.api.cx.metamask.io';
```

Comment:
Should the name of this file reflect the API itself so `accounts-api.ts`?

I've also added this same util (for a different endpoint) for the ongoing incoming transactions refactor, so should we match the name and location to simplify the merge?

https://github.com/MetaMask/core/blob/8b7874cde7e15281638e53170ee0e47329f1c9da/packages/transaction-controller/src/api/accounts-api.ts

Reply:
Good call Matthew, adjusted the all things to sync with your changes